src/query/config/src/config.rs

```diff
@@ -430,7 +430,7 @@ pub struct CatalogsHiveConfig {
 #[serde(default)]
 pub struct HiveCatalogConfig {
     #[clap(long = "hive-meta-store-address", default_value_t)]
-    pub address: String,
+    pub hive_metastore: String,
```

Comment:
The same.

Reply:
This is also a compatible change: `impl FromToProto for HiveCatalogConfig` will use the same field name.